### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by elkinmilen**
Addin initial version of README.md

### CODE REVIEW

**Comment by elkinmilen** [comment]
@nelci592 Please review and also ensure I have properly described the content by chapters.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+[Telerik Reporting video course](https://learn.telerik.com/learn/course/internal/view/elearning/38/telerik-reporting)
+available in [Telerik's Virtual Classroom](https://learn.telerik.com/learn).
+
+After the Welcome chapter, the second chapter makes an overview of what tools the 
+[Telerik Reporting](https://www.telerik.com/products/reporting?utm_medium=referral&utm_source=github&utm_campaign=reporting-awareness-gh-public-readme)
+product includes and what a report definition is.
+
+Chapter 3 guides you in the process of designing a basic report using the
+[Standalone Report Designer](https://docs.telerik.com/reporting/designing-reports/report-designer-tools/desktop-designers/standalone-report-designer/overview)
+and integrating it into a sample Blazor application.
```
Comment (nelci592) [inline README.md]: ... into a sample Blazor application by using the Blazor Report Viewer ?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Chapter 4 shows some more advanced scenarios of report customization and also shows how to work with the alternative 
+[Web Report Designer](https://docs.telerik.com/reporting/designing-reports/report-designer-tools/web-report-designer/overview)
+that is used to embed report creation into your web applications.
+[Globalization](https://docs.telerik.com/reporting/globalization/overview) of the reports is also covered.
+
+Chapter 5 teaches how to better style your reports.
```
Comment (nelci592) [inline README.md]: ... your reports by creating style rules and setting different style properties of the report items.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Both active trialists and license holders can take advantage of our legendary technical support provided directly by the Reporting dev team!
+
+## **Useful Links**
+
+- [Video onboarding training Telerik Reporting](https://learn.telerik.com/learn/course/internal/view/elearning/38/telerik-reporting)
+- [Initial version of the RPS Issue Tracker Projects GitHub Repo](https://github.com/telerik/RpsTrackerBlazorReportingStart)
+- [Basic Final version of the RPS Issue Tracker Projects GitHub Repo](https://github.com/telerik/RpsTrackerBlazorReportingStart/tree/basics-complete)
+- [Advanced Final version of the RPS Issue Tracker Projects GitHub Repo](https://github.com/telerik/RpsTrackerBlazorReportingStart/tree/advanced-complete)
+- Browse all [Telerik Reporting live demos](https://demos.telerik.com/reporting?utm_medium=referral&utm_source=github&utm_campaign=reporting-awareness-gh-public-readme)
```
Comment (nelci592) [inline README.md:58]: maybe online demos?

**Review by nelci592 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (elkinmilen) [reply README.md]: Good idea. Added.

File: README.md (hunk 2 above)
Comment (elkinmilen) [reply README.md]: Added!

File: README.md (hunk 3 above)
Comment (elkinmilen) [reply README.md:58]: Already available. Thanks.

**Pull request merged**